### ISSUE TITLE
The training pipeline and training time of this work

### ISSUE THREAD

**Issue by linhuixiao** [microsoft/RegionCLIP#50]
The training pipeline and training time of this work

1. I understand that the paper uses the Resnet and text transformer of CLIP for initialization, and then uses the region-text pair extracted from CC3M for pre-training, and then splices the pre-trained model and RPN together and trains the RPN on the base class, right?

2. CC3M has 3 million pictures. How many GPUs are used for pre-training and how much training time is consumed? This is not mentioned in the paper.

Thanks.

**Comment by YiwuZhong** [comment]
1. Yes. The visual backbone is initialized by CLIP and further pre-trained by region-text pairs. The RPN is trained by the boxes of base classes, yet not using categorical labels.
2. You can refer to the pre-training script [here](https://github.com/microsoft/RegionCLIP/blob/main/pretrain.sh). For ResNet50, it takes roughly 6 days using 32 V100 GPUs.

**Comment by linhuixiao** [comment]
Thanks a lot for your kind reply.
